riotfile.py

```diff
@@ -638,7 +638,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         ),
         Venv(
             name="cherrypy",
-            command="python -m pytest {cmdargs} tests/contrib/cherrypy",
+            command="pytest --no-ddtrace {cmdargs} tests/contrib/cherrypy",
```

Comment:
Pulling these `--no-ddtrace` changes out into a separate pull request would clarify to reviewers why they're necessary and clearly prove that they don't break existing tests.

Reply:
Good point, will take a look at this. Using `--no-ddtrace` might not be needed after all